react/controls/TrackControl.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { RefObject, useRef, FunctionComponent, ChangeEvent } from 'react'
+
+import styles from '../styles/styles.css'
+import useVideoTrack from '../useVideoTrack'
+
+export interface TrackControlProps {
+  videoRef: RefObject<HTMLVideoElement>
+  changeState: (property: string, value: any) => void
+  cssHandles: any
+}
+
+const TrackControl: FunctionComponent<TrackControlProps> = ({
+  videoRef,
+  changeState,
+  cssHandles,
+}) => {
+  const trackBarRef = useRef<HTMLInputElement>(null)
+  const { duration, currentTime, setCurrentTime, formatTime } = useVideoTrack(
+    videoRef,
+    trackBarRef
+  )
+
+  const handleChange = (event: ChangeEvent) => {
+    if (videoRef?.current?.ended) {
+      changeState('isPlaying', false)
+    }
+
+    const target = event?.target as HTMLInputElement
+
+    setCurrentTime(parseFloat(target?.value))
+  }
+
+  return (
+    <div
+      className={`${cssHandles.trackContainer} w-100 absolute bottom-1 lh-title`}
+    >
+      {currentTime && duration && (
+        <span className={`${cssHandles.trackTimer} ml7 white v-mid`}>
+          {formatTime(currentTime)} / {formatTime(duration)}
+        </span>
+      )}
+
+      <input
+        ref={trackBarRef}
+        onChange={(event) => {
+          handleChange(event)
+        }}
+        className={`${cssHandles.trackBar} ${styles.trackBar} w-100 seekbar v-mid`}
```

Comment:
Do we need the class `seekbar`?
```suggestion
        className={`${cssHandles.trackBar} ${styles.trackBar} w-100 v-mid`}
```

Reply:
I think that's because of the custom CSS, right? There are two ways of creating a CSS Handle:

1. Using useCssHandles hook
2. Using a class from a CSS Module

So, if you're using this to style it, you might just change the method of adding a CSS handle to the second option (using CSS Module) and add the custom styling there.

Reply:
I forgot it there hehe